tools/sof-dump-status.py

```diff
@@ -186,9 +186,11 @@ def _getPowerPath(device_id):
 
         for pci_info in self.pci_lst:
             if 'device_id' in pci_info:
-                exit_code, output = subprocess.getstatusoutput("cat %s" % (_getPowerPath(pci_info['device_id'])))
-                if exit_code == 0:
-                    self.sys_power['run_status'].append({'map_id': pci_info['device_id'], 'status': output})
+                #exit_code, output = subprocess.getstatusoutput("cat %s" % (_getPowerPath(pci_info['device_id'])))
```

Comment:
Tested on Helios, BSW, BDW and BYT, all passed. `_getPowerPath(device_id):`  may not be used anymore, you can delete it safely.